src/components/dialogs/parameters/widget/parameter-float.tsx

```diff
@@ -14,24 +14,35 @@ export interface ParameterFloatProps {
     label: string;
     style: any;
     adornment: any;
+    tooltip?: string;
 }
 
 export const ParameterFloat = ({
     name,
     label,
     style,
     adornment,
+    tooltip,
 }: ParameterFloatProps) => {
     return (
         <>
-            <Grid item container direction={'row'} spacing={1} paddingTop={3}>
-                <Grid item xs={10} sx={style}>
-                    <FormattedMessage id={label} />
-                </Grid>
-                <Grid item xs={2}>
-                    <FloatInput name={name} adornment={adornment} />
+            <Tooltip title={tooltip ? <FormattedMessage id={tooltip} /> : ''}>
```

Comment:
don't put Tooltip component if tooltip is not defined ?

Reply:
Done